src/react-demo/Content/public/index.html

```diff
@@ -8,7 +8,7 @@
     <link rel="shortcut icon" type="image/png" href="img/favicon-32x32.png" sizes="32x32" />
     <link rel="shortcut icon" type="image/png" href="img/favicon-16x16.png" sizes="16x16" />
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css" />
-    <script src="https://cdn.polyfill.io/v2/polyfill.js?features=default,fetch"></script>
+    <script src="https://cdn.polyfill.io/v2/polyfill.js?features=es6"></script>
```

Comment:
Shouldn't it be `es6,fetch`. That's what I'm using in the simple template.

Reply:
Hum, the template doesn't include Fable.PowerPack and don't use fetch so for me it's find like that.

Reply:
Elmish has a dependency on Fable.PowerPack and to users it would be a bit difficult to realize they need to add `fetch` here if they want to use the `Fetch` module. At a minimum I would add a note in the README :)

Reply:
Ah yes true. I will try to think to do it tomorrow. (adding the fetch polyfill)